assets/js/blocks/cart-checkout-shared/use-forced-layout.ts

```diff
@@ -116,6 +115,7 @@ export const useForcedLayout = ( {
 
 	const registry = useRegistry();
 	useEffect( () => {
+		let templateSynced = false;
```

Comment:
Wouldn't `templateSynced` get reset on every re-render, or is the `useRegistry` here stable?

Reply:
It seems stable, logging this happens only once when a block is rendered for the first time.